fiftyone/utils/annotations.py

```diff
@@ -957,6 +962,9 @@ def load_annotations(
     Args:
         samples: a :class:`fiftyone.core.collections.SampleCollection`
         anno_key: an annotation key
+        destination_field (None): an optional name of a new destination field
```

Comment:
Please add this parameter to `SampleColleciton.load_annotations()` too, for completeness